src/lib/secrets_helper.ts

```diff
@@ -133,20 +82,46 @@ const formatSecrets = (result: OneGraphSecretsResponse | undefined) => {
   return newSecrets
 }
 
-type OneGraphPayload = { authlifyToken: string | undefined }
+interface RequestHeaders {
```

Comment:
It would be good to add a comment indicating that this is the [Web API Headers type](https://developer.mozilla.org/en-US/docs/Web/API/Headers)